includes/custom-meta/_custom-meta.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+// Style those forms a bit
+function admin_style() {
+  wp_enqueue_style('admin-styles', get_template_directory_uri().'/includes/custom-meta/meta-box-styles.css');
```

Comment:
I'd just add an `admin.scss` next to `main.scss` and import all admin styles into that.